core/communication/port-authority/src/util.ts

```diff
@@ -34,7 +32,7 @@ export type Try<T> = Success<T> | Failure;
  * See [[recoverPromise]] for the inverse operation.
  */
 export async function rethrowPromise<T>(t: Try<T>): Promise<T> {
```

Comment:
Note - This does not look to be a `rethrow`, it is more like boxing/mapping Try to Promise.